### PR TITLE
fixup sns topic subscriptions

### PR DESCRIPTION
**Pull request by nand0p**
##### ISSUE TYPE
- Bugfix Pull Request
##### COMPONENT NAME

cloud/amazon/sns_topic
##### ANSIBLE VERSION

```
ansible 1.9.5
```
##### SUMMARY

Currently, sns_topic does not remove subscriptions on destroy, leaving subscription orphans to linger.
This PR cleans up subscriptions, if they exist, prior to deleting topic.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @nand0p. To the current maintainers, @joelthompson please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by joelthompson** [comment]
LGTM. shipit

**Comment by gregdek** [comment]
Thanks again to @nand0p for this PR, and thanks @joelthompson for reviewing. Marking for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/amazon/sns_topic.py [hunk 1]
```diff
@@ -97,35 +97,34 @@
 """
 
 RETURN = '''
-topic_created:
-    description: Whether the topic was newly created
+changed:
```
Comment (resmo) [inline cloud/amazon/sns_topic.py]: changed is a common return and must not be added in returns doc. Please remove it.

**Comment by nand0p** [comment]
@resmo updated

**Pull request merged**